app/src/main/java/com/arygm/quickfix/model/profile/Profile.kt

```diff
@@ -0,0 +1,10 @@
+package com.arygm.quickfix.model.profile
+
+data class Profile(
+    val uid: String,
+    val firstName: String,
+    val lastName: String,
+    val email: String,
+    val password: String,
+    val birthDate: com.google.firebase.Timestamp
+)
```

Comment:
As discussed during the meeting, we should add a boolean to differentiate the user from the worker, or create another model for the worker.

Reply:
This will come in a future PR (I will be working on it 😄)